app/models/institution_tree.rb

```diff
@@ -38,43 +37,40 @@ def main_campus(institution)
       return institution if institution.campus_type == 'Y' || institution.campus_type.nil?
 
       str = <<-SQL
-        WITH RECURSIVE related_up AS(
-          SELECT campus_type, facility_code, parent_facility_code_id
-          FROM institutions WHERE facility_code = ? AND version = ?
-          UNION
-            SELECT
-              i.campus_type, i.facility_code, i.parent_facility_code_id
-            FROM institutions i
-            INNER JOIN related_up r ON r.parent_facility_code_id = i.facility_code
-            WHERE i.version = ?
-        ) SELECT * FROM related_up where campus_type = 'Y'
+      WITH RECURSIVE related_up AS(
+        SELECT campus_type, facility_code, parent_facility_code_id
+        FROM institutions WHERE facility_code = ?
```

Comment:
I think you're still going to want the `version` clause:
```suggestion
        FROM institutions WHERE facility_code = ? AND version_id = ?
```